md_api/src/main/java/com/wipro/ats/bdre/md/api/Import.java

```diff
@@ -1,5 +1,7 @@
 package com.wipro.ats.bdre.md.api;
 
+import org.apache.log4j.Logger;
+
 import java.io.*;
 import java.util.List;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)